### DISCUSSION TITLE
No pricing information is available for this part.

### DISCUSSION THREAD

**Discussion by rob-dm-c** [inventree/InvenTree#3412] [Q&A] [answered]
When viewing the pricing tab, or using the "show pricing information" button, no supplier pricing is displayed. Error:
No pricing information is available for this part.

It does not pick up the price set in the stock item. The only way we can get it to display a price is by manually setting a Supplier Price Break in the admin view.

Has anyone got this working who can give me some pointers on what I may be doing wrong?

**Comment by matmair** [comment]
@robIDI have you enabled the pricing features in the global settings?
Stock prices and supplier prices are two seperate things btw. If pricing is enabled in the instance and on the part they will show up in separate categories.

**Comment by rob-dm-c** [comment, marked as answer]
@matmair Thanks for your reply.

 I think i have enabled all the pricing features, but I'm very new to this system so I could have missed something. I cannot see any pricing features specifically for supplier parts. Is it Supplier Part Breaks? If so, is the only way to add them in the Admin View? It could very well be that I am just misunderstanding. 
 
 Either way, it wasn't supplier part I needed in the end :)

I wanted to get the total price of a build order, the BOM gives us that information so we're good on that from.

Thanks a lot anyway

**Reply by matmair** [reply]
@robIDI great that you found what you needed.

The supplier price breaks are set in the supllier parts, see screenshot / link:
https://demo.inventree.org/supplier-part/203/#
![grafik](https://user-images.githubusercontent.com/66015116/181353500-b9ad3555-b7e0-48be-8eb0-8bb4237de9ff.png)

**Reply by rob-dm-c** [reply]
@matmair Thanks for the screenshot. I have no idea how I missed that in my search!
